lib/auth.coffee

```diff
@@ -215,6 +215,41 @@ getAuth = (deps, opts) ->
 		.catchReturn(false)
 		.asCallback(callback)
 
+	###*
+	# @summary Generate a new API key for user
+	# @name generateApiKey
+	# @public
+	# @function
+	# @memberof resin.auth
+	#
+	# @description This will only work if you user is logged in.
```

Comment:
I'm not sure what exactly this means. Do you mean 'only work if you are logged in with a JWT'? I don't think we need to say that you need to be logged in at all for this method to work - that's true for 90% of methods in the SDK.

Also, the description should also mainly explain what this method does: it registers a new api key for the current user with the name given.